workloads/gcf.go

```
@@ -34,3 +40,35 @@ func Push() error {
 	err := Cf("push", "pats-"+guid.String(), "patsapp", "-m", "64M", "-p", "assets/hello-world").ExpectOutput("App started")
 	return err
 }
+
+func MoveAndSalt(srcDir string, dstDir string, salt string){
+	filepath.Walk(srcDir,  func(file string, info os.FileInfo, err error) error { 
+		if err != nil { panic(err) }
+		pathTail := strings.SplitAfter(file, srcDir)[1]
+		if info.IsDir() {
+			os.Mkdir(path.Join(dstDir, pathTail), 0777)
+		} else if info.Mode().IsRegular() {
+			input, _ := ioutil.ReadFile(file)
```

for both this line, and L54, better to return the error instead of hiding it, then the workload can fail with a helpful reason